netbox/extras/tests/test_custom_field_choice_sets_endpoint.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+
+from users.models import Token
+
+User = get_user_model()
+
+
+class CustomFieldChoiceSetsEndpointTest(APITestCase):
```

Comment:
All tests for this endpoint should go in `extras.tests.test_api.CustomFieldChoiceSetTest`.